### ISSUE TITLE
THREE.ObjectLoader.parseObject; a case for LineSegments missing

### ISSUE THREAD

**Issue by bGute** [mrdoob/three.js#7684]
Within the switch case in the ObjectLoader.parseObject function is a case for the "LineSegements" missing. By now a serialized LineSegments gets deserialized into a Object3D. [see here](https://github.com/mrdoob/three.js/blob/dev/src/loaders/ObjectLoader.js#L539)

**Comment by mrdoob** [comment]
Oh noes... 

Hmm, actually, we recently added `setDrawMode()` to `Mesh`.

https://github.com/mrdoob/three.js/blob/dev/src/objects/Mesh.js#L26-L30

It would probably make more sense to follow that with `Line` too (and remove `LineSegments` 😓)...

```javascript
var line = new THREE.Line( geometry, material );
line.setDrawMode( THREE.LinesDrawMode ); // default: THREE.LineStripDrawMode
```

@WestLangley sounds good?

**Comment by WestLangley** [comment]
It's reasonable, but it seems to me that `drawMode` should be a property of the geometry -- not the mesh or line.

How do we know if a geometry represents a sequence of points, or a sequence of pairs of points? I recall this being an issue in #7013, and elsewhere.

**Comment by WestLangley** [comment]
I would keep `THREE.LineSegments` as an extension of `THREE.Line` if possible. Personally, I prefer that API. But it would be nice to have a property of the geometry that identifies how the data is stored in `THREE.Geometry` and `THREE.BufferGeometry`.

**Comment by Mugen87** [comment]
Fixed via #9334
